src/core_modules/capture-core/components/WidgetEventEdit/EditEventDataEntry/editEventDataEntry.epics.js

```diff
@@ -163,6 +163,15 @@ export const saveEditedEventEpic = (action$: InputObservable, store: ReduxStore)
 export const saveEditedEventSucceededEpic = (action$: InputObservable) =>
     action$.pipe(
         ofType(actionTypes.EDIT_EVENT_DATA_ENTRY_SAVED),
+        filter((action) => {
+            const {
+                meta: { triggerAction },
+            } = action;
+            return (
+                triggerAction === enrollmentSiteActionTypes.COMMIT_ENROLLMENT_EVENT ||
+                triggerAction === enrollmentEditEventActionTypes.EVENT_SAVE_ENROLLMENT_COMPLETE_SUCCESS
+            );
+        }),
```

Comment:
Is this really needed? Seems like we return EMPTY on line 183. Is it good enough to do one or the other?

Reply:
On my localhoast, the `return EMPTY` statement is causing an error and the page is no longer responding when editing a single event. I remove it and filter by `triggerAction` instead. This is not an issue in play so should I just rollback the change and leave it as it is? 🤔 

<img width="2324" alt="Screenshot 2024-04-04 at 17 54 42" src="https://github.com/dhis2/capture-app/assets/6817073/a2178cb0-3137-4b4c-95a3-badfbf4af819">

Thanks!

Reply:
I think the reason `EMPTY` crashes the app is that it counts as an observable and not an action; so it works as return value for `switchMap` but not for `map`. I think using `filter` is the best solution here. I approve the last commit :+1:

Reply:
I see, thanks. Current PR looks good to me.